src/Platform/Microsoft.Testing.Platform/OutputDevice/Terminal/TerminalTestReporter.cs

```diff
@@ -980,4 +990,24 @@ private static TerminalColor ToTerminalColor(ConsoleColor consoleColor)
             ConsoleColor.White => TerminalColor.White,
             _ => TerminalColor.Default,
         };
+
+    public void TestInProgress(
+        string assembly,
+        string? targetFramework,
+        string? architecture,
+        string testNodeUid,
+        string displayName,
+        string? executionId)
+    {
+        TestProgressState asm = _assemblies[$"{assembly}|{targetFramework}|{architecture}|{executionId}"];
+
+        if (_options.ShowActiveTests)
+        {
+            asm.TestNodeResultsState ??= new(_counter++);
+            asm.TestNodeResultsState.AddRunningTestNode(
+                _counter++, testNodeUid, $"{displayName}", CreateStopwatch());
```

Comment:
Nit: we don't need the interpolated string.

```suggestion
                _counter++, testNodeUid, displayName, CreateStopwatch());
```